docs/reference/coordinates/index.rst

```diff
@@ -84,246 +84,7 @@ see `Thompson (2006) <https://doi.org/10.1051/0004-6361:20054262>`_
 and `Franz & Harper (2002) <https://doi.org/10.1016/S0032-0633(01)00119-2>`_
 (and `corrected version <https://www2.mps.mpg.de/homes/fraenz/systems/systems3art/systems3art.html>`_).
```

Comment:
Ha I had the exact opposite reaction when looking at the table. I would say it is solidly "reference" material, and very much not "how-to-guide" material. From [the diátaxis on how-to guides](https://diataxis.fr/how-to-guides/), how-to guide characteristics include:
- action and only action
- no digression, explanation, teaching

Reply:
I disagree.  If the table had the classes listed in the first column, sorted by the class names, and the other information in subsequent columns, it'd be a reference table.  However, as formatted, sorted, and intended to be used, I argue that it is a bare how-to guide.  Perhaps it'd be clearer if the column titles were renamed:

- "Coordinate system" -> "If you want to use the coordinate system..."
- "Abbreviation" -> "also abbreviated as..."
- "SunPy/Astropy equivalent" -> "use the class..."
- "Notes" -> "but be aware that..."

Reply:
Ok I can see the logic in how this *could* be used as a how-to guide and if that was how it was intended to be used, we should promote it as such. A relabeling of the columns as suggested would definitely make that more clear. Maybe breaking it out of a table and/or adding a title and description akin to "coordinate transform recipes" might also help?

I still think this could be useful piece of reference material as well. During the course of trying to understand what coordinate systems sunpy supports, a user may want to be able to look up what the supported coordinate systems are and a table is a natural way to present that. They should not need to go to a how-to guide to find that information.

Reply:
I think that as it is currently written this table should be a reference. If we want to reformat it later and add it to the guide then we should, but I think we should try and keep large copy editing to followup PRs?

Reply:
I agree this seems like reference, so I'll leave it here in this PR. Open to someone else convincing us that it should be moved to another section in a further PR.

Reply:
Well, even if we can't agree on where the table should be if were to be in only one place, don't implement the two tables with duplicated RST, because that's just annoying from a maintenance perspective.  The table should live as a separate RST snippet that is inserted in both places using the `include` directive.